WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -191,16 +191,20 @@ extension ReaderTabView {
         guard filterButton.titleLabel?.text == "Filter" else {
             return
         }
-        setFilterButtonTitle("Phoebe's Photos")
-        resetFilterButton.isHidden = false
-        viewModel.presentFilter()
+        viewModel.presentFilter() { [weak self] topic in
+            if let topic = topic {
```

Comment:
would it be possible to leave the view agnostic of any model (in this case ReaderAbstractTopic)? More details in the next comment